test/statement/close.js

```diff
@@ -124,7 +122,12 @@ describe('.close([calback])...', () => {
         });
       });
     });
-    it('...should close after calling prepare with an error (bad sql prepared).', (done) => {
+    it('...should close after calling prepare with an error (bad sql prepared).', function(done) {
+      // SQL Server doesn't check for syntax error when the application calls SQLPrepare
+      if (global.dbms === 'mssql')
+      {
+        return this.skip('aix!');
```

Comment:
aix?

Reply:
Ahh yeah, I was trying to see if the skip function would print when skipped (it doesn't), and I think it was during our meeting and the last words I heard were "aix".